torch_np/_ufuncs.py

```diff
@@ -107,12 +129,14 @@ def divmod(
     if out1.shape != out2.shape or out1.dtype != out2.dtype:
         raise ValueError("out1, out2 must be compatible")
 
-    tensors = _helpers.ufunc_preprocess(
-        (x1, x2), out, True, casting, order, dtype, subok, signature, extobj
+    tensors = _ufunc_preprocess(
+        (x1, x2), True, casting, order, dtype, subok, signature, extobj
     )
 
-    result = _binary_ufuncs_impl.divmod(*tensors)
+    quot, rem = _binary_ufuncs_impl.divmod(*tensors)
 
+    quot = _ufunc_postprocess(quot, out, casting)
+    rem = _ufunc_postprocess(rem, out, casting)
```

Comment:
These should be out1 and out2 respectively.

Reply:
Thanks, indeed.